index.php

```diff
@@ -30,7 +30,7 @@
 
       if (!empty($link_post_id)) {
 
-        $categories = get_categories($link_post_id);
+        $categories = wp_get_object_terms($link_post_id, 'category');
```

Comment:
`It should be noted that the results from wp_get_object_terms are not cached which will result in a db call everytime this function is called. For performance, functions like get_the_terms() (which the results of has been cached), should be used.`

can `get_the_terms` be used instead?